src/Coordination/KeeperSnapshotManager.cpp

```diff
@@ -635,7 +635,11 @@ nuraft::ptr<nuraft::buffer> KeeperSnapshotManager::serializeSnapshotToBuffer(con
 
 bool KeeperSnapshotManager::isZstdCompressed(nuraft::ptr<nuraft::buffer> buffer)
 {
+#if __BYTE_ORDER__ == __ORDER_BIG_ENDIAN__
```

Comment:
constexpr unsigned char magic[4] = "\xAA\xBB..."

Reply:
@alexey-milovidov, I modified the fix accordingly, please review.